ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -546,6 +579,64 @@ def test_nginx_commits_after_remove(tmpdir, settings, staticsite_path,
     assert manager().get_service.call_count == 2
     manager().restart.assert_not_called()
 
+pytestmark = pytest.mark.django_db
```

Comment:
Doesn't that mark all the tests in this module?

Since most don't require the db to be set up, wouldn't it be preferrable to only explicitly mark the functions requiring it?